.github/ISSUE_TEMPLATE/Special-Interest-Group-Contribution.md

```diff
@@ -1,106 +1,157 @@
+
+Instructions and a template for you to adjust are avaialble in the [Contribution Process Socialization](https://community.finos.org/docs/governance/software-projects/contribution/#step-2-socialization) documentation.
+
+- [ ] I confirm I have socialized this contribution to `community@finos.org` (to be checked after the issue is raised)
+
+# 2. Approval
+The [FINOS Governing Board]([https://github.com/finos/technical-oversight-committee](https://www.finos.org/governing-board)) is responsible for approving new FINOS standards.
+
+Once you are happy with the socialization phase (garnered enough support / interest, addressed open questions, etc), please prepare a short slide deck (10m), check the box below, and put a comment on this issue so the proposal can be sent to the Governing Board for approval via email or at the next Governing Board meeting 
+
+- [ ] I am happy for this issue to be sent to the Governing Board for formal review and approval and I prepared a short SIG proposal deck for the FINOS Governing Board `replace [this template](/governance-docs/202103-New-Project-or-SIG-proposal-(template)-PUBLIC.pptx) with link or attach to the issue` 
+- [ ] FINOS team notified leadership@finos.org to trigger Governing Board approval and assigned this issue to COO
+- [ ] FINOS Leadership confirmed timing and procedure of Governing Board vote with contributors
+
+### Governing Board Approval findings 
+- [ ] FINOS Governing Board reviewed the contribution and the contribution was: `accepted|rejected`
+
+> FINOS team to enter high level findings summary here supporting the decision
+
+## 3. Preparing For Onboarding
+Before the FINOS team can onboard your project, there are a few housekeeping items that need to be taken care of from an IP and infastructure standpoint. These must be completed by the contributor, with help if required from the FINOS Infrastructure team, so please familiarize yourself with the [requirements](/docs/governance/software-projects/contribution/#step-4-ip-review--transfer-of-contributions).
+
+### Steps
+- [ ] Usable name and logo identified (either existing or new)
+  - [ ] FINOS team confirmed project brand and logo are usable in the public domain. This might require completion of [project contribution agreement](https://community.finos.org/governance-docs/The.Linux.Foundation.--.Form.of.Trademark.Assignment.20221202.pdf) to properly assign trademarks.
+  - [ ] Failed the above, new project name and logo is identified
+- [ ] FINOS team enabled documentation DCO (developer certificate of origin) to enforce CC-by-40 license, [default for SIGs](/docs/governance#special-interest-groups)
```

Comment:
DCO cannot be enabled until after we control the repo. As such this should happen after the code transfer.